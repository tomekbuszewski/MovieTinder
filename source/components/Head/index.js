import React from 'react';
import { Helmet } from 'react-helmet';
import PropTypes from 'prop-types';

/**
 * Component for generating <head> section
 * @param {string} title - title of the page
 */
// @flow
const Head = ({ title }: { title: string }) => <Helmet>
  <title>{title} — BeerGuru — An example app</title>
  <meta property="og:title" content={`${title} — BeerGuru — An example app`} />
</Helmet>;

Head.propTypes = {
  title: PropTypes.string.isRequired
};

export default Head;
