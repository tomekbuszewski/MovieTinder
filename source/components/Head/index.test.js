import React from 'react';
import renderer from 'react-test-renderer';

import Head from './index';

describe ('Head tests', () => {
  it ('should match snapshot', () => {
    const tree = renderer.create(<Head title="Some title" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
