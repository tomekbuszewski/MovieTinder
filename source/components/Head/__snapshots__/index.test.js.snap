// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head tests should match snapshot 1`] = `null`;
